packages/mobile/App.svelte

```diff
@@ -25,19 +25,23 @@
     } from 'shared/routes'
     import { onMount } from 'svelte'
 
+    $mobile = true
```

Comment:
`mobile` is of [writable](https://svelte.dev/tutorial/writable-stores) type. We should either `update` it or `set` it.

Reply:
I will use set or update if to you is more readable. Also since the "mobile" store is auto-subscribed in the html code, Svelte allows to use the $ prefix in the <script> tag also for assignments, for the sake of readability I think.
The only difference is on compiled code where Svelte import set_store_value instead of simply use mobile.set